.hashibot.hcl

```diff
@@ -1550,41 +1558,41 @@ behavior "pull_request_path_labeler" "service_labels" {
 }
 
 behavior "regexp_issue_labeler" "panic_label" {
-    regexp = "panic:"
```

Comment:
might be an IDE issue here causing non changing changes

Reply:
Good catch! I actually was about to change them back but then found out the new format matches the top indention for the same blocks. Now, I am confused

Reply:
BTW, I rolled-back this change anyway.